crates/ruff_dev/src/generate_all.rs

```diff
@@ -21,7 +21,7 @@ pub(crate) enum Mode {
     /// Don't write to the file, check if the file is up-to-date and error if not.
     Check,
 
-    /// Write the generated help to stdout (rather than to `docs/configuration.md`).
+    /// Write the generated help to stdout
```

Comment:
Nit: don't kill me but trailing periods on all user-facing docs please (here and crates/ruff_dev/src/generate_options.rs and crates/ruff_dev/src/generate_rules_table.rs) 😅

Reply:
good to know, will do